### PR TITLE
Update prettyjson to 1.1.x

### PR DESCRIPTION
**Pull request by dbushong**
- It now displays multiline blocks in """ ... """

I'm not sure if the new look is better or not, but the old `prettyjson` had an `engine` requirement section that produces warnings on Node 4.x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "format-error",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (EndangeredMassa) [inline package.json]: Can you undo this change? I prefer to bump this in an explicit commit before publish.

**Comment by dbushong** [comment]
(updated)
